packages/ui/src/2_molecules/VerticalTabs/components/VerticalTabItem.module.css

```diff
@@ -0,0 +1,28 @@
+.button {
+  @apply w-full block text-left text-gray-10 text-opacity-50 font-medium font-roboto;
+  &.active {
+    @apply text-opacity-100;
+  }
+
+  & p {
+    @apply text-base leading-[1.171875rem];
```

Comment:
``1.171875rem`` value is very custom. i think we should ask designer not to use it